server/app/views/questiontypes/ApplicantQuestionRendererImpl.java

```diff
@@ -99,7 +99,8 @@ public final DivTag render(ApplicantQuestionRendererParams params) {
     ContainerTag questionTag;
     ImmutableList<DomContent> questionTextDoms =
         TextFormatter.createLinksAndEscapeText(
-            question.getQuestionText(), TextFormatter.UrlOpenAction.NewTab);
```

Comment:
Also the screenshots will need to be updated https://docs.civiform.us/contributor-guide/developer-guide/testing#screenshot-diffing

Reply:
Done.